Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -2502,6 +2502,16 @@ static HTMLFormElement* scanForForm(Element* start)
     return nullptr;
 }
 
+static ValidatedFormListedElement* findFormControlElementAncestor(Element* element)
+{
+    while (element) {
```

Comment:
You can use lineageOfType<Element>(*element) here.

Reply:
Neat, [f5f2c2c](https://github.com/WebKit/WebKit/pull/4988/commits/f5f2c2c1992d14428931fa2f084e8a63b968b15b).